mu4/scenes/palette/qml/MuseScore/Palette/PaletteBackground.qml

```diff
@@ -28,7 +28,7 @@ Canvas {
     property int cellWidth: 24
     property int cellHeight: 24
 
-    property color background: ui.theme.backgroundColor //! TODO mscore.paletteBackground
+    property color background: ui.theme.backgroundPrimaryColor //! TODO mscore.paletteBackground
     property color gridColor: enabled ? "#808080" : "#55808080"
```

Comment:
Can we replace these codes with colors from the theme?

Reply:
https://github.com/musescore/MuseScore/pull/6471#discussion_r473030881